# Type: refactor
# Subtype: R8
# English Summary: Move the field to the top of the class
# Chinese Summary: 将字段移动到类的顶部
# Entry Number: 13

# === Git Diff Content ===

diff --git a/WordPress/src/main/java/org/wordpress/android/ui/prefs/SettingsFragment.java b/WordPress/src/main/java/org/wordpress/android/ui/prefs/SettingsFragment.java
index 5f75cf878..9ff61128a 100644
--- a/WordPress/src/main/java/org/wordpress/android/ui/prefs/SettingsFragment.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/prefs/SettingsFragment.java
@@ -60,6 +60,7 @@ import java.util.Map;
 public class SettingsFragment extends PreferenceFragment {
     public static final String SETTINGS_PREFERENCES = "settings-pref";
 
+    private AlertDialog mDialog;
     private SharedPreferences mSettings;
     private WPEditTextPreference mTaglineTextPreference;
 
@@ -387,7 +388,6 @@ public class SettingsFragment extends PreferenceFragment {
         }
     };
 
-    private AlertDialog mDialog;
     private final OnPreferenceClickListener languagePreferenceClickListener = new OnPreferenceClickListener() {
         @Override
         public boolean onPreferenceClick(Preference preference) {
